docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -94,3 +97,11 @@ Camunda 8 data produced from these resources will belong to the same tenant. The
 example on how tenant ownership is inherited.
 
 ![Tenant ownership inheritance diagram](img/multi-tenancy.png)
+
+## Unsupported features
+
+Multi-tenancy only works for Self-Managed installations with authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/).
+
+For a standalone Zeebe instance, multi-tenancy is currently available with authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/)
+or by [integrating an external tenant-managing component](../zeebe-deployment/zeebe-gateway/interceptors.md#implementing-a-tenant-providing-interceptor)
```

Comment:
❓ Why the title is `Unsupported features` ?

Reply:
Good point. What I added is not really an unsupported feature. Maybe a separate section called "Standalone Zeebe usage" would work better.

I'm open for other suggestions for the heading :smile:

Reply:
Yeah, but what is the point to repeat what wrote at lines 32-34 ? 😄 
If we want to add a specific section for standalone zeebe, I will remove the lines 32-34 and create a separate section `Standalone Zeebe usage`, but without the Self-Managed explanation

Reply:
Although it's not supported for the full Camunda 8 stack. I'll see how I can rephrase a mention in this section.